.github/workflows/dependent-issues.yml

```
@@ -0,0 +1,36 @@
+---
+name: Dependent Issues
+
+on:
+  issues:
+    types:
+      - opened
+      - reopened
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+  pull_request:
+    types:
+      - opened
+      - reopened
+  schedule:
+    - cron: '0 * * * *' # check hourly
```

```suggestion
    - cron: '0 0 * * *' # check daily
```

Based on @dfarrell07 comments  ^ but checking daily.

---

Commented as above and quoting again

Yes, I modified the issue/PR trigger on open and re-open. But I feel daily is still too long a time gap. However, I want to get this merged ASAP. So going with daily checks for now. We can always edit it later.

---

Agreed, we can increase the check frequency as we use the feature more.